goblet/tests/test_bqremotefunction.py

```diff
@@ -19,11 +19,15 @@ def string_test_blogs_1(x: str, y: str) -> str:
 
         resources = app.handlers["bqremotefunction"].resources
 
-        input, output = BigQueryRemoteFunction._get_hints(string_test_blogs_1)
```

Comment:
it would be nice to add a new test case for this feature. i got one to work below. 

```
    def test_call_bqremotefunction_vectorize(self, monkeypatch):
        monkeypatch.setenv("GOOGLE_PROJECT", "TEST_PROJECT")
        monkeypatch.setenv("GOOGLE_LOCATION", "us-central1")

        test_name = "bqremotefunction_test"
        app = Goblet(function_name=test_name)

        @app.bqremotefunction(dataset_id="blogs",vectorize_func=True)
        def function_test(x: List[int], y: List[str]) -> List[str]:
            return [f"{x[0]}{y[0]}",f"{x[1]}{y[1]}"]

        body = {
            "userDefinedContext": {
                "X-Goblet-Name": "bqremotefunction_test_function_test"
            },
            "calls": [[2, "a"], [3, "b"]],
        }

        mock_request = Mock()
        mock_request.json = body
        mock_request.headers = {}

        result = json.loads(app(mock_request, None))

        assert result["replies"][0] == "2a"
        assert result["replies"][1] == "3b"
```

Reply:
thank you! added that in